### ISSUE TITLE
Confusion in versions & docs

### ISSUE THREAD

**Issue by brdloush** [walkable-server/walkable#169]
Hello,

I'm looking into walkable and am wondering what's the project's status? Is is still being maintained and is it considered usable for production use? 

While trying to learn the basics and trying to run the EQL queries in my REPL, I found some issues with versioning. I'm not sure which version and which documentation should I use..

- for both `1.2.0` and `1.3.0` there only seems to be [SNAPSHOT versions present on clojars](https://clojars.org/walkable/versions), not a "non-snapshot" version
- README.md references `[walkable "1.2.0-SNAPSHOT"]` dependency
- [https://walkable.gitlab.io/ page](https://walkable.gitlab.io) points to [documentation of version 1.3.0](https://walkable.gitlab.io/walkable/1.3.0/introduction.html). There doesn't seem to be documentation for `1.2.0` available on gitlab.io page
  - But the actual content of that `1.3.0` documentation (such as [Installation part](https://walkable.gitlab.io/walkable/1.3.0/installation.html)) still refers to `[walkable "1.2.0-SNAPSHOT"]`
  - similarly, the doc is referencing `walkable.sql-query-builder` namespace, but in `1.3.0-SNAPSHOT` the namespace (file `sql_query_builder.cljc`) seems no longer to exist.

Thank you for clarification.

**Comment by myguidingstar** [comment]
hi @brdloush Thanks for sharing your experience and sorry for the confusion. Version 1.2.0-SNAPSHOT has been around for a while and I've developed major features around it. It would be stable in terms of intended behavior, but I can't say the public api and terminologies are bearable. I started 1.3.0-SNAPSHOT as a fix for that plus supporting Pathom Connect, but ended up rewriting a lot. The good news is they look good with the new design. I will release a new version soon.
TLD; version: v1.2.* works but is kinda ugly and won't get long-term supported. Effort is being spent on a better api and hence newer version. Until then, no production grade is confirmed.
That being said, you're welcome to ask questions regarding v1.2.* in the project slack channel.
